openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/unityCatalogConnection.json

```diff
@@ -0,0 +1,104 @@
+      },
+      "hostPort": {
+        "title": "Host and Port",
+        "description": "Host and port of the Databricks service.",
+        "type": "string"
+      },
+      "token": {
+        "title": "Token",
+        "description": "Generated Token to connect to Databricks.",
+        "type": "string",
+        "format": "password"
+      },
+      "httpPath": {
+        "title": "Http Path",
+        "description": "Databricks compute resources URL.",
+        "type": "string"
+      },
+      "catalog": {
+        "title": "Catalog",
+        "description": "Catalog of the data source(Example: hive_metastore). This is optional parameter, if you would like to restrict the metadata reading to a single catalog. When left blank, OpenMetadata Ingestion attempts to scan all the catalog.",
+        "type": "string"
+      },
+      "databaseSchema": {
+        "title": "Database Schema",
+        "description": "Database Schema of the data source. This is optional parameter, if you would like to restrict the metadata reading to a single schema. When left blank, OpenMetadata Ingestion attempts to scan all the schemas.",
+        "type": "string"
+      },
+      "connectionTimeout": {
+        "title": "Connection Timeout",
+        "description": "The maximum amount of time (in seconds) to wait for a successful connection to the data source. If the connection attempt takes longer than this timeout period, an error will be returned.",
+        "type": "integer",
+        "default": 120
+      },
+      "connectionOptions": {
+        "title": "Connection Options",
+        "$ref": "../connectionBasicType.json#/definitions/connectionOptions"
+      },
+      "connectionArguments": {
+        "title": "Connection Arguments",
+        "$ref": "../connectionBasicType.json#/definitions/connectionArguments"
+      },
+      "supportsUsageExtraction": {
+        "$ref": "../connectionBasicType.json#/definitions/supportsUsageExtraction"
+      },
+      "supportsLineageExtraction": {
+        "$ref": "../connectionBasicType.json#/definitions/supportsLineageExtraction"
+      },
+      "supportsDBTExtraction": {
+        "$ref": "../connectionBasicType.json#/definitions/supportsDBTExtraction"
+      },
+      "supportsMetadataExtraction": {
+        "title": "Supports Metadata Extraction",
+        "$ref": "../connectionBasicType.json#/definitions/supportsMetadataExtraction"
+      },
+      "supportsDatabase": {
+        "title": "Supports Database",
+        "$ref": "../connectionBasicType.json#/definitions/supportsDatabase"
+      },
+      "supportsQueryComment": {
+        "title": "Supports Query Comment",
+        "$ref": "../connectionBasicType.json#/definitions/supportsQueryComment"
+      },
+      "sampleDataStorageConfig": {
+        "title": "Storage Config for Sample Data",
+        "$ref": "../connectionBasicType.json#/definitions/sampleDataStorageConfig"
+      }
+    },
+    "additionalProperties": false,
```

Comment:
Note:  Have removed the supportProfiler from unity catalog connector as we have this ticket open #12947